program1.py

```diff
@@ -0,0 +1,77 @@
+"""
+this is first program to counte the number of vowels, words, sentences, and punctuation marks.
+"""
+
+from importlib.resources import contents
+from string import punctuation
+
+
+
+def count_vowels(test_Sentence):
+    number_of_vowels=0
+    """
+    this function is used to count vowels
```

Comment:
Every developer knows that this is a function. Can we change it like
`Counts the number of vowels in a sentence`